bin/build-monorepo-file.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to build docs/site MONOREPO.md file with filling
+ * up all packages with lexicographically order
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+const Project = require('@lerna/project');
+const createMarkdownTable = require('markdown-table');
+
+const MONOREPO_FILE_DIST = 'docs/site';
+const MONOREPO_FILE_NAME = 'MONOREPO-draft.md';
+
+async function getSortedPackages() {
+  const project = new Project(process.cwd());
+  const packages = await project.getPackages();
+  packages.sort((p1, p2) => p1.name.localeCompare(p2.name));
+  return packages;
+}
+
+function getPackageFields(pkg) {
+  const packageJson = fs.readJsonSync(path.join(pkg.location, 'package.json'));
+
+  const relativeUri = path.relative(pkg.rootPath, pkg.location);
+  const pkgUrl = `https://github.com/strongloop/loopback-next/tree/master/${relativeUri}`;
+
+  return [
+    `[${relativeUri}](${pkgUrl})`,
+    pkg.private ? '_(private)_' : pkg.name,
+    packageJson.description,
+  ];
+}
+
+function buildTable(packages) {
+  const header = ['Package', 'npm', 'Description'];
+
+  return createMarkdownTable([header, ...packages.map(getPackageFields)]);
+}
+
+async function buildMonorepoFile() {
+  const packages = await getSortedPackages();
+  const markdownTable = buildTable(packages);
+
+  const content = [
+    '# Monorepo overview',
+    '',
+    'The [loopback-next](https://github.com/strongloop/loopback-next) repository uses',
+    '[lerna](https://lernajs.io/) to manage multiple packages for LoopBack 4.',
+    '',
+    '<!-- PLEASE KEEP THE TABLE ROWS SORTED ALPHABETICALLY BY PACKAGE NAME -->',
```

Comment:
Now that the content of this file is auto-generated, we should change this comment to ask people to use your new script to update the contents in this file. The sorting no longer matters in this comment, because the script will take care of keeping consistent order.